src/Nethermind/Nethermind.Runner.Test/ConfigFilesTests.cs

```diff
@@ -225,20 +224,20 @@ public void IsMining_enabled_for_ndm_consumer_local(string configWildcard)
             Test<IInitConfig, bool>(configWildcard, c => c.IsMining, true);
         }
 
-        [TestCase("baseline", true)]
-        [TestCase("spaceneth", true)]
-        [TestCase("^baseline ^spaceneth", false)]
-        public void Baseline_is_disabled_by_default(string configWildcard, bool enabled)
-        {
-            Test<IBaselineConfig, bool>(configWildcard, c => c.Enabled, enabled);
-        }
-
-        [TestCase("ndm", true)]
-        [TestCase("^ndm", false)]
-        public void Ndm_enabled_only_for_ndm_configs(string configWildcard, bool ndmEnabled)
-        {
-            Test<INdmConfig, bool>(configWildcard, c => c.Enabled, ndmEnabled);
-        }
+        // [TestCase("baseline", true)]
+        // [TestCase("spaceneth", true)]
+        // [TestCase("^baseline ^spaceneth", false)]
+        // public void Baseline_is_disabled_by_default(string configWildcard, bool enabled)
+        // {
+        //     Test<IBaselineConfig, bool>(configWildcard, c => c.Enabled, enabled);
+        // }
+
+        // [TestCase("ndm", true)]
+        // [TestCase("^ndm", false)]
+        // public void Ndm_enabled_only_for_ndm_configs(string configWildcard, bool ndmEnabled)
+        // {
+        //     Test<INdmConfig, bool>(configWildcard, c => c.Enabled, ndmEnabled);
+        // }
```

Comment:
?

Reply:
ok, I think i have just found a way of testing it

Reply:
fixed